### PR TITLE
SPDK charts and examples

### PR DESCRIPTION
**Pull request by derekbit**

### CODE REVIEW

File: examples/spdk/storageclass.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+metadata:
+  name: longhorn-spdk
+provisioner: driver.longhorn.io
+allowVolumeExpansion: true
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+parameters:
+  numberOfReplicas: "2"
+  staleReplicaTimeout: "2880"
+  fromBackup: ""
```
Comment (innobead) [inline examples/spdk/storageclass.yaml:12]: If not supported yet, we can comment out. Also, add a comment next to `parameters` to mention those parameters not supported yet.

File: chart/templates/crds.yaml [hunk 2]
```diff
@@ -1772,6 +1782,8 @@ spec:
                   properties:
                     spec:
                       properties:
+                        backendStoreDriver:
```
Comment (innobead) [inline chart/templates/crds.yaml:1785]: Do we need this for deprecated `instances`?

**Review by innobead — CHANGES_REQUESTED** [review]
In general, LGTM. Some comments needed to resolve.

File: chart/templates/crds.yaml (hunk 2 above)
Comment (derekbit) [reply chart/templates/crds.yaml:1785]: Yes, instances in old IMs need the field as well.

File: examples/spdk/storageclass.yaml (hunk 1 above)
Comment (derekbit) [reply examples/spdk/storageclass.yaml]: Addressed, thank you.

**Comment by innobead** [comment]
Pending until manager PRs merged

File: chart/templates/crds.yaml [hunk 3]
```diff
@@ -1678,6 +1684,8 @@ spec:
...
```
Comment (shuo-wu) [inline chart/templates/crds.yaml:1687]: Why do not some `backendStoreDriver` fields include `enum`?

Comment (derekbit) [reply chart/templates/crds.yaml:1687]: Not sure if we need to include `enum` for the `backendStoreDriver` are in status.

Comment (shuo-wu) [reply chart/templates/crds.yaml:1687]: If it's allowed and there is compatibility issue, I think it's better to have

**Pull request merged**